web/tailwind.config.cjs

```diff
@@ -19,6 +19,9 @@ const config = {
           },
         },
       },
+      spacing: {
+        15: '3.75rem',
+      },
```

Comment:
Haha nice, I was just about to suggest this actually 😂

Reply:
![b23a908ae01021bc1064937bad061b11](https://github.com/user-attachments/assets/935a95db-e648-4330-8b3c-7032bd458475)